hw/ip/rv_dm/data/rv_dm.hjson

```diff
@@ -103,15 +103,144 @@
     regs:[
     ]
     mem: [
-      // Debug memory size (given as `items` below) must be a power of two.
+      // Expansion of debug memory within rv_dm used for DV purposes. The generated reg_top is not
+      // actually instantiated in RTL since the debug memory region is implemented inside the
+      // vendored-in module from the PULP project.
+      //
+      // The debug memory region is specified in the PULP debug system documentation here:
+      // https://github.com/pulp-platform/riscv-dbg/blob/master/doc/debug-system.md
+      //
+      // The debug module exposes a 16kB memory called debug memory. It has a ROM portion
+      // (debug ROM), some memory mapped CSRs and a RAM portion (program buffer). This region is
+      // accessible over the TL interface only if debug mode is active.
+      //
+      // TODO: Most of the CSRs below are required to be parameterized to the number of Harts the
+      // debug module caters to, which in case of RV_DM is 1. Consider making this actually
+      // parameterized to more accurately reflect the state of the design. The address skips and
```

Comment:
We could address this comment here by parameterizing the defs below with `NrHarts` value, right?

Reply:
Good point - let me try that!

Reply:
Ok I parameterized the registers that are multiregs inside the RV_DM. This only affected `HALTED` and `RESUMING`.